### PR TITLE
Now we know what this repo does!

### PR DESCRIPTION
**Pull request by lachellel**
Updated descriptions and removed idevity tags

### CODE REVIEW

**Comment by lachellel** [comment]
@konklone

Does this readme make more sense?

File: README.md [hunk 1]
```diff
@@ -1,4 +1,34 @@
...
+
+No warranties made on prettiness of the outputs!  Screenshot sample:
+
+![Screenshot of HTML Output](/screenshot_html_output.PNG)
+
+## Using
+
+* Deploy as war in J2EE application server (i.e. Apache Tomcat)
+* Tested on Tomcat 7
+* Please note: the CRL files WILL be fetched. Some of these files are greater than 30 MB.  Rapid testing or iterative refreshes may impact I/O costs.
```
Comment (grandamp) [inline README.md]: Refreshing the URL that displays the validation cache has no effect.  To reset the cache (at the moment) you need to restart the app.

File: README.md [hunk 2]
```diff
@@ -1,4 +1,34 @@
...
+* Uses discovered objects from SIA (down from root) and recurses to perform AIA from those discovered objects
+* Outputs all certificates discovered in PEM form, and an HTML log showing results including timing transactions for P7B(AIA), OCSP(AIA) and CRL(CDP) URIs identified in the certificates
+
+No warranties made on prettiness of the outputs!  Screenshot sample:
+
+![Screenshot of HTML Output](/screenshot_html_output.PNG)
+
+## Using
+
+* Deploy as war in J2EE application server (i.e. Apache Tomcat)
```
Comment (grandamp) [inline README.md:30]: Or, run directly in eclipse by right clicking on the project, and select "run on server" using a Tomcat 7+ plugin.

**Comment by lachellel** [comment]
updated for first item

**Comment by konklone** [comment]
> @konklone
> 
> Does this readme make more sense?

Yes, it makes a ton more sense!

**Comment by grandamp** [comment]
I've opened issues to provide background, and what I intend to do.  Any
input is appreciated!

**Pull request merged**